st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -305,6 +305,12 @@
                 'type': 'boolean',
                 'default': False
             },
+            'verify_ssl_cert': {
+                'description': 'SSL verification is not supported.',
+                'type': 'boolean',
+                'default': False,
+                'immutable': True
```

Comment:
So does this mean we are giving an option to verify SSL cert but not an option to pass a cacert bundle?